call-home/src/common/constants.rs

```diff
@@ -1 +1,19 @@
+/// PRODUCT is the name of the project for which this call-home component is deployed.
 pub(crate) const PRODUCT: &str = "Mayastor";
+
+/// DATA_DIR_PATH is the directory path for the temporary files generated during
+/// encryption.
+pub(crate) const DEFAULT_DATA_DIR_PATH: &str = "/encryption_dir";
```

Comment:
Shouldn't this default to a local dir? How do we run this locally?

Reply:
Done. This now defaults to the present working directory.

Reply:
The key path still seems to be in the root. 
I also don't see a key, will that be added later?
What about tests?